src/app/MessageDef/ReportDataMessage.h

```diff
@@ -86,7 +86,7 @@ class Parser : public MessageParser
      *  @return #CHIP_NO_ERROR on success
      *          #CHIP_END_OF_TLV if there is no such element
      */
-    CHIP_ERROR GetSubscriptionId(uint64_t * const apSubscriptionId) const;
+    CHIP_ERROR GetSubscriptionId(uint32_t * const apSubscriptionId) const;
```

Comment:
Should this be SubscriptionId instead of uint32_t?

Reply:
Created https://github.com/project-chip/connectedhomeip/pull/18410 for SubscriptionId stuff.